README.md

```diff
@@ -1,52 +0,0 @@
-# People Fall Detection
```

Comment:
@bhavikapanara why delete the README.md. It needs to be filled out with more relevant content.